build.gradle

```diff
@@ -14,30 +23,19 @@ buildscript {
 }
 
 group "com.trusona.integration"
-version "1.3.3-SNAPSHOT"
+version "1.4.0-SNAPSHOT"
```

Comment:
Should this be a major version bump to coincide with the FR major version bump?

Reply:
like `2.0.0`? I can bump it higher